<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="WIN-39319556043" failures="1" tests="18" name="com.rs.test.configuration.carrierService.CarrierService" time="422.375" errors="0" timestamp="2022-07-04T13:01:13 CST" skipped="0">
  <testcase classname="com.rs.test.configuration.carrierService.CarrierService" name="editCarrierService" time="24.379"/>
  <system-out/>
  <testcase classname="com.rs.test.configuration.carrierService.CarrierService" name="filters" time="36.155"/>
  <system-out/>
  <testcase classname="com.rs.test.configuration.carrierService.CarrierService" name="createRoutePoint" time="22.945"/>
  <system-out/>
  <testcase classname="com.rs.test.configuration.carrierService.CarrierService" name="createCarrierService" time="20.112"/>
  <system-out/>
  <testcase classname="com.rs.test.configuration.carrierService.CarrierService" name="editRoutePoint" time="29.806"/>
  <system-out/>
  <testcase classname="com.rs.test.configuration.carrierService.CarrierService" name="deleteRoutePoint" time="20.873"/>
  <system-out/>
  <testcase classname="com.rs.test.configuration.carrierService.CarrierService" name="createLogAgent" time="24.759"/>
  <system-out/>
  <testcase classname="com.rs.test.configuration.carrierService.CarrierService" name="deleteLogAgent" time="21.568"/>
  <system-out/>
  <testcase classname="com.rs.test.configuration.carrierService.CarrierService" name="createMOChar" time="26.819"/>
  <system-out/>
  <testcase classname="com.rs.test.configuration.carrierService.CarrierService" name="editMOChar" time="25.506"/>
  <system-out/>
  <testcase classname="com.rs.test.configuration.carrierService.CarrierService" name="deleteMOChar" time="20.455"/>
  <system-out/>
  <testcase classname="com.rs.test.configuration.carrierService.CarrierService" name="editMSGProps" time="0.001">
    <failure message="This page is not automated yet, due to a bug expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: This page is not automated yet, due to a bug expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at com.rs.test.base.TestBaseSteven.pending(TestBaseSteven.java:727)
at com.rs.test.pageObjectModel.configuration.carrierService.CarrierServicePO.editMSGProps(CarrierServicePO.java:1723)
at com.rs.test.configuration.carrierService.CarrierService.editMSGProps(CarrierService.java:158)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- editMSGProps -->
  <system-out/>
  <testcase classname="com.rs.test.configuration.carrierService.CarrierService" name="createMTChar" time="27.755"/>
  <system-out/>
  <testcase classname="com.rs.test.configuration.carrierService.CarrierService" name="createMSGProps" time="32.844"/>
  <system-out/>
  <testcase classname="com.rs.test.configuration.carrierService.CarrierService" name="deleteMSGProps" time="21.641"/>
  <system-out/>
  <testcase classname="com.rs.test.configuration.carrierService.CarrierService" name="editMTChar" time="28.303"/>
  <system-out/>
  <testcase classname="com.rs.test.configuration.carrierService.CarrierService" name="deleteMTChar" time="21.475"/>
  <system-out/>
  <testcase classname="com.rs.test.configuration.carrierService.CarrierService" name="deleteCarrierService" time="16.979"/>
  <system-out/>
</testsuite> <!-- com.rs.test.configuration.carrierService.CarrierService -->
